pictures/templatetags/pictures.py

```diff
@@ -43,7 +43,12 @@ def picture(field_file, img_alt=None, ratio=None, container=None, **kwargs):
             "alt": img_alt,
             "ratio": (ratio or "3/2").replace("/", "x"),
             "sources": sources,
-            "media": utils.sizes(container_width=container, **breakpoints),
+            "media": utils.sizes(
+                cols=field_file.field.grid_columns,
+                container_width=container,
+                breakpoint_settings=field_file.field.breakpoints,
+                **breakpoints,
+            ),
```

Comment:
I don't think you need to pass the field breakpoints here, do you?

Reply:
the issue is precisely not passing the field settings and instead using global settings. If you have more than one PictureField in a project with different settings they both end up using the global settings because the field settings are not passed to various utility functions